### ISSUE TITLE
Unable to create policy

### ISSUE THREAD

**Issue by stepan111** [minio/minio#11559]
In recent minio release I am not able to create a policy. I had to use minio version `RELEASE.2020-10-28T08-16-50Z-20-g38f12c984` and minio/mc version `RELEASE.2021-02-07T02-02-05Z` to create custom policy.

## Expected Behavior
Policy would be created

## Current Behavior
minio client failed with next message:
```
mc: <ERROR> Unable to parse the input policy: conditions are not supported for action s3:DeleteBucket.
```

## Possible Solution
Downgrade client to version  `RELEASE.2021-02-07T02-02-05Z`  and minio server to version `RELEASE.2020-10-28T08-16-50Z-20-g38f12c984`

## Steps to Reproduce (for bugs)
1. Create policy file
```
cat >test <<EOF                                                                                    
{
  "Version": "2012-10-17",
  "Statement": [
    {
      "Sid": "Stmt1",
      "Action": "s3:*",
      "Effect": "Allow",
      "Resource": "arn:aws:s3:::minio-test/*"
    },
    {
      "Sid": "Stmt2",
      "Action": [
        "s3:CreateBucket",
        "s3:DeleteBucket"
      ],
      "Effect": "Deny",
      "Resource": "arn:aws:s3:::*"
    }
  ]
}
EOF

```
2. Apply policy to minio
```
mc admin policy add minio test ./test                                                                


```

## Context
It broke my deployment pipeline(

## Regression
No

## Your Environment
I am using minio as proxy for GCS.

**Comment by harshavardhana** [comment]
Fixed by #11546 in master

**Comment by harshavardhana** [comment]
Duplicate of #11544
